dipy/denoise/meson.build

```diff
@@ -25,7 +25,7 @@ python_sources = ['__init__.py',
   'nlmeans.py',
   'noise_estimate.py',
   'non_local_means.py',
-  'patch2self.py',
+  'patch2self.py'
```

Comment:
Why is this removed ? If this is a necessary style change, it should go into a separate style commit.